.github/workflows/deploy.yml

```diff
@@ -110,6 +111,7 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           path: ./dist/qiskit*
+          name: aer-deploy-sdist-${{ matrix.os }}
       - name: Publish package distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Deployed right after - maybe give a `aer-deploy-separate-` prefix to avoid being pulled by the shared uploader?

Reply:
Added the prefix as you suggested in d9d7fb78